src/core/crypto/impl/cryptopp/elgamal.cc

```diff
@@ -163,5 +164,25 @@ void GenerateElGamalKeyPair(
 #endif
 }
 
+// Create deterministic keypair
+void GenerateDeterministicElGamalKeyPair(
+    std::uint8_t* priv,
+    std::uint8_t* pub,
+    std::uint8_t* seed) {
+#if defined(__x86_64__) || defined(__i386__) || defined(_MSC_VER)
```

Comment:
ARM build fails. If you really want deterministic keypair, you should simply remove this definition and DH keypair generation.

Reply:
Was wondering about the necessity of the preprocessor macros here, I'll remove them.